honeybee_energy/construction.py

```diff
@@ -36,8 +41,9 @@ def __init__(self, name, materials):
                 Can include spaces but special characters will be stripped out.
             materials: List of materials in the construction (from outside to inside).
         """
+        self._locked = False  # unlocked by default
```

Comment:
As we discussed, it is set by the `lockable` decorator but there's a (very slight) performance improvement if we specify what `_locked` variable we want in `__init__`.  Also, what you said yesterday about `__slots__` is not correct.  Just because there is a slot available for an attribute doesn't mean that `hasattr` will return True.  The slot actually has to be filled with a value for it `hasattr` to return True.  So the very slight performance improvement of specifying the `_locked` variable still holds here.

Reply:
Thanks.